internal/tiltfile2/tiltfile_state.go

```diff
@@ -309,6 +317,32 @@ func (s *tiltfileState) translate(resources []*k8sResource) ([]model.Manifest, e
 	return result, nil
 }
 
+func (s *tiltfileState) translateDC(resources []dcResource) ([]model.Manifest, error) {
+	if len(resources) == 0 {
+		return nil, nil
+	}
+
+	// TODO(maia): support more than one docker-compose file.
```

Comment:
i'm not sure this is worth implementing until we know someone needs it? it seems like it'd be easier to emit a good error message at the dockerCompose call site, so that the error message gets appropriate line numbers, etc

Reply:
aah yeah, good call.